### PR TITLE
Add node.js version of script

### PR DESCRIPTION
**Pull request by South-Paw**
* Added node.js version of the script
* Updated readme to be even easier to ... read ..?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +29,17 @@
...
+### Python
 
 0. (optional) Setup virtual env: `virtualenv env && source env/bin/activate`
 1. `pip install requests tqdm`
 2. Run the script: `python cheat.py`
+
+### Node.js
+
+1. Install the _Latest_ [Node.js](https://nodejs.org/en/) version
+2. Open the `node` folder and then open command line
```
Comment (SinZ163) [inline README.md]: None of the other languages go this verbose in steps, why go the distance here?

**Comment by xPaw** [comment]
I think we're better off if you just keep your own version in a separate repository.

I've changed the logic multiple times already in PHP version, and Python version isn't 100% in sync with that. Adding yet another version into that won't help anyone.

File: README.md (hunk 1 above)
Comment (South-Paw) [reply README.md]: I was discussing with some people on discord and some of them couldn't work it out how to use it without help.

I'm happy to go overboard on other explanations as well if needbe - just want to make it accessible.

Comment (xPaw) [reply README.md]: You could make a separate PR to expand our readme though :)

**Comment by South-Paw** [comment]
I'm happy to have a shot and re-syncing them again tomorrow morning - otherwise happy to keep it separate as well.

File: README.md (hunk 1 above)
Comment (South-Paw) [reply README.md]: Fair point!

**Comment by South-Paw** [comment]
@xPaw your call on what you'd like to do with this 👍

**Comment by xPaw** [comment]
Yeah I think it's just easier to keep it separate.

Do feel free to make a PR to expand the readme though.

**Comment by South-Paw** [comment]
Righto, lets close this and I'll make a readme PR tomorrow 💤

**Pull request closed without merging**